include/brickengine/input_keycodes.hpp

```diff
@@ -0,0 +1,63 @@
+#ifndef FILE_INPUT_KEYCODES_HPP
+#define FILE_INPUT_KEYCODES_HPP
+
+#include "SDL2/SDL.h"
+
+enum InputKeyCodes {
```

Comment:
Consider using enum class: https://stackoverflow.com/questions/18335861/why-is-enum-class-preferred-over-plain-enum